main_he.tex

```diff
@@ -43,18 +50,19 @@
 
 \def\sections{sections/translated/he}
 \def\lang_header_adjustment{0px}
+\newtoggle{hommrtl}
+\toggletrue{hommrtl}
```

Comment:
Can we use something more descriptive? I understand in the context of this PR "rtl" to mean "right to left", but after a while I won't remember what it was. I suggest `textright2left`.

Reply:
I don't mind that, but I don't see the point. RTL for Right to Left is very common (for those who use right to left languages at least...)
we can also add a comment in the metadata, if that helpful

Reply:
It is obvious to you, yes, but I've never seen that in my life before. Believe me, quite a lot of people who worked on this never dealt with RTL text. Please give it a self-explanatory name, not a comment. Those extra few characters cost you nothing, but could mean the world for others.

Reply:
As I said, I really don't mind that.

Fixed.